src/Storage/Storage/DatalakeGen2/Cmdlet/UpdateAzDataLakeGen2Item.cs

```diff
@@ -53,9 +53,11 @@ public class SetAzDataLakeGen2ItemCommand : StorageCloudBlobCmdletBase
         [ValidateNotNull]
         public AzureDataLakeGen2Item InputObject { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Sets POSIX access permissions for the file owner, the file owning group, and others. Each class may be granted read, write, or execute permission. Symbolic (rwxrw-rw-) is supported. Invalid in conjunction with ACL.")]
+        [Parameter(Mandatory = false, HelpMessage = "Sets POSIX access permissions for the file owner, the file owning group, and others. Each class may be granted read, write, or execute permission. Symbolic (rwxrw-rw-) is supported." +
+            "The sticky bit is also supported and its represented either by the letter t or T in the final character-place depending on whether the execution bit for the others category is set or unset respectively, absence of t or T indicates sticky bit not set." +
+            "Invalid in conjunction with ACL.")]
         [ValidateNotNullOrEmpty]
-        [ValidatePattern("([r-][w-][x-]){3}")]
+        [ValidatePattern("([r-][w-][x-]){2}[r-][w-][xtT-]")]
         public string Permission { get; set; }
```

Comment:
Per [SDK](https://github.com/Azure/azure-sdk-for-net/blob/ae550cb5d4be459e61d0f42f46c00033b0fc9a83/sdk/storage/Azure.Storage.Files.DataLake/src/Models/PathPermissions.cs#L116) , we should allow an optional "+" in the end for ExtendedAcls .
Please help to modify it to `[r-][w-][x-]){2}[r-][w-][xtT-][+]{0,1}`

Reply:
Done

Reply:
I just test the suffix "+" looks not work on server.
And it's also not in our rest API, so we might should not add it now.
Would you please help to revert it by remover "[+]{0,1}"?